fm-demo/feedSetup.js

```diff
@@ -40,8 +40,13 @@ async function registerFeedCreatorIfNeeded(api, aliceAccount, operatorAccount) {
 }
 async function createFeed(api, sender) {
     console.log(`Creating feed with config: ${JSON.stringify(feedConfig, null, 4)}`);
+
+    // make the pallet admin also the oracle admin
+    const palletAdmin =  await api.query.chainlinkFeed.palletAdmin();
+    feedConfig.oracles = feedConfig.oracles.map(oracle => [oracle, palletAdmin]);
```

Comment:
@mattsse let's make the oracle admin same as the oracle, same as the original intention

```suggestion
    feedConfig.oracles = feedConfig.oracles.map(oracle => [oracle, oracle]);
```